octobot_trading/modes/channel/abstract_mode_producer.py

```diff
@@ -76,8 +81,16 @@ def __init__(self, channel, config, trading_mode, exchange_manager):
         # Define trading modes default consumer priority level
         self.priority_level: int = channel_enums.ChannelConsumerPriorityLevels.MEDIUM.value
 
-        self.run_dbs_identifier = None
         self.symbol = None
+        self.reload_config()
+        self._is_first_call = True
+
+    def reload_config(self):
```

Comment:
If I understand well it should be named `on_reloaded_config` instead of `reload_config`

Reply:
Right, I'm changing that